lib/api/github.js

```diff
@@ -51,6 +51,7 @@ async function initRepo(repoName, token, endpoint) {
     logger.error(`GitHub init error: ${JSON.stringify(err)}`);
     throw err;
   }
+  return config;
```

Comment:
It could be a good idea to update the `gitlab` equivalent for consistency's sake, even if it's not tested yet.